tt_metal/python_env/requirements-dev.txt

```diff
@@ -57,4 +57,6 @@ fsspec==2023.9.2 # Temporary pin to 2023.9.2: https://github.com/tenstorrent/tt-
 docopt==0.6.2
 tabulate==0.9.0
 blobfile==2.1.1 # Required for llama3
+pydantic==2.9.2 # Required for llama3
```

Comment:
we really need to find ways to create separate dependencies suites for different use cases
If this is only needed for llama, there should be a separate pipenv the user creates

Reply:
Which file is pydantic used? I don't see a reference to it

Reply:
it's required by `models/demos/llama3/reference/llama_models`

Reply:
why is it listed here then?

Reply:
you can install the pip wheel before you run the test in the GH workflow. That would remove the need for you deal with dependencies completely right?